doc/source/projects.rst

```diff
@@ -0,0 +1,79 @@
+Ray Projects (Experimental)
+===========================
+
+Ray projects make it easy to package a Ray application so it can be
+rerun later in the same environment. They allow for the sharing and
+reliable reuse of existing code.
+
+Quick start (CLI)
+-----------------
+
+.. code-block:: bash
+
+    # Creates a project in the current directory. It will create a
+    # project.yaml defining the code and environment and a cluster.yaml
+    # describing the cluster configuration. Both will be created in the
+    # .rayproject subdirectory of the current directory.
+    $ ray project create <project-name>
+
+    # Create a new session from the given project with an optional name.
+    # Launch a cluster and run the appropriate command.
+    $ ray session start [--name="default-session"]
```

Comment:
these optional arguments are not implemented in the CLI PR. because multiple session snapshot is not implemented. Can we remove them in doc and add these back when they are implemented? 

```diff
- ray session start [--name="default-session"]
+ ray session start 

- ray session attach [<default-session>] [--start]
+ ray session attach 

- ray session stop [<default-session>] [--terminate]
+ ray session stop
```